src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -746,11 +746,45 @@ private Result ConstructPathTraversalResult(ContrastLogReader.Context context)
+
+            // default : The web page '{0}' had 'ValidateRequest' set to 'false' in the page directive. Request Validation helps prevent several types of attacks including XSS by detecting potentially dangerous character sequences. An exception is thrown by the framework when a potentially dangerous character sequence is encountered. This exception returns an error page to the user and prevents the application from processing the request. An attacker can submit malicious data to the application that may be processed without further input validation. This malicious data could contain XSS or other injection attacks that may have been prevented by ASP.NET request validation. Note that request validation does not provide 100% protection against XSS or other attacks and should be thought of as a defense-in-depth measure.
+
+            // <properties name="aspx">\WebGoatCoins\ProductDetails.aspx</properties>
+            // <properties name="snippet">1: &lt;%@ Page Title="" Language="C#" ValidateRequest="false" ... %&gt;</properties>
+
+            IDictionary<string, string> properties = context.Properties;
+            string aspx = properties[nameof(aspx)];
+            string snippet = properties[nameof(snippet)];
+
+            Result result = CreateResultCore(context);
+            result.Locations = new List<Location>
+            {
+                new Location
+                {
+                    PhysicalLocation = CreatePhysicalLocation(aspx, CreateRegion(snippet)),
+                }
+            };
+
+            result.Message = new Message
+            {
+                Id = "default",
+                Arguments = new List<string>
+                {              // The web page 
+                    aspx       // '{0}' had 'ValidateRequest' set to 'false' in the page directive. ...
+                }
+            };
+
+            return result;
+        }
+
         private Result ConstructRequestValidationModeDisabledResult(ContrastLogReader.Context context)
         {
             // request-validation-control-disabled : Request Validation Mode Disabled
 
-            // default : The configuration in '{0}' had 'ValidateRequest' set to 'false' in the page directive. Request Validation helps prevent several types of attacks including XSS by detecting potentially dangerous character sequences. An exception is thrown by the framework when a potentially dangerous character sequence is encountered. This exception returns an error page to the user and prevents the application from processing the request. An attacker can submit malicious data to the application that may be processed without further input validation. This malicious data could contain XSS or other injection attacks that may have been prevented by ASP.NET request validation. Note that request validation does not provide 100% protection against XSS or other attacks and should be thought of as a defense-in-depth measure.
```

Comment:
>The [](start = 25, length = 3)

This is the confusion I mentioned in the PR description. request-validation-control-disabled looks at the control level (not at the page level) and it looks for a `ValidateRequestMode` attribute (not a `ValidateRequest` attribute) with value `Disabled` (not `false`).